### DISCUSSION TITLE
Is the Xamarin.Forms.Maps Control going to be migrated to Maui?

### DISCUSSION THREAD

**Discussion by mkdave** [dotnet/maui#4573] [Q&A]
Is the Xamarin.Forms.Maps Control going to be migrated to Maui?

I see an entry called Map under the Maui/Wiki/Status page under the heading Other Handlers that has a status of pending.

Is this the Control equivalent to Xamarin.Forms.Maps?

**Comment by Brosten** [comment]
I'm also looking for an answer on this question. Seems the control didn't make it to the GA. But what are the plans now? Any idéa of if and in that case _when_ it will be available in MAUI?

**Comment by hartez** [comment]
It's on [the roadmap](https://github.com/dotnet/maui/wiki/Roadmap), but no firm dates at the moment.

**Reply by hartez** [reply]
As of right now, we are targeting November (and .NET 7) for porting Xamarin.Forms.Maps to .NET MAUI. As with everything, this timing is subject to change.

**Reply by dmccolloughOneGas** [reply]
Is there a technical reason that it's not going to be until .NET 7?
We have a couple of apps and both use the Map control so until it's released we are not going to be able to update the apps to Maui.

Thanks

**Reply by hartez** [reply]
More scheduling/administrative reasons than technical ones. Hopefully we'll have some version of Maps in a preview before the final release, so you'll be able to start working on updates before then. But again, no guarantees.

**Reply by TonyLugg** [reply]
I'm also stuck in Xamarin.Forms until Maps is available.  Seems short-sighted to be so delayed.

**Reply by hartez** [reply]
It's available in the .NET 7 previews if you'd like to give it a try: https://devblogs.microsoft.com/dotnet/dotnet-maui-in-dotnet-7-rc1/#maps
